### DISCUSSION TITLE
Repository permissions doesn't explain that they're constrained by org permissions

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#21061] [Actions]
I have an org that has locked down default permissions for repositories.

I don't remember that detail, because I set it a while ago, but it could be an org I don't actively control, and I could just have permissions to do things to a repository...

So, I visit the repository settings because a workflow I'm trying to use is getting readonly tokens and it's breaking. This repository is a fork of another organization's repository where the same workflow works.

Here's what I see:

<img width="788" alt="image" src="https://user-images.githubusercontent.com/2119212/180402492-2524754c-ee01-422a-b185-38d6c151e558.png">

### Workflow permissions
Choose the default permissions granted to the GITHUB_TOKEN when running workflows in this repository. You can specify more granular permissions in the workflow using YAML. [Learn more.](https://docs.github.com/actions/reference/authentication-in-a-workflow#modifying-the-permissions-for-the-github_token)

⭕ Read and write permissions
Workflows have read and write permissions in the repository for all scopes.

🔘 Read repository contents permission
Workflows have read permissions in the repository for the contents scope only.

----

This is not a great user experience. The read and write permissions item is disabled and there's no explanation. There is a link that talks about the direct settings, but it's a couple of links away from the actual documentation that matters:

https://docs.github.com/en/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization#configuring-the-default-github_token-permissions

----

## Expected results

If some other setting is preventing a user from making a change, the UI should include an explanation and a link to the relevant setting for *that* setting -- not just the documentation for the thing the user can't change.

**Comment by mickael-h** [comment]
Thank you for this!
I had been pulling my hair for almost an hour trying to understand why I could not change the permission for my own workflows on my own private repository in my own organization XD

**Reply by jsoref** [reply]
I filed https://github.com/github/docs/issues/29271
